config/configmap.yaml

```diff
@@ -24,7 +24,6 @@ data:
       "powerBiUrl": "",
       "gaId": "",
       "features": {
-        "newDarUi": true,
-        "newChairConsole": false
+        "newDarUi": true
```

Comment:
I don't think we need this value either. I believe this was the feature for the new voting screens for a DAR.

Reply:
this comment and the one above on base_config seem conflicting, do we want to get rid of the new DarUI flag because we aren't using it anywhere or add it to the other configs so they're consistent?

Reply:
@raejohanek - You are correct, I am in conflict with myself. I think removing `newDarUi` altogether is right answer.